ss_admin_console/main/default/classes/constants/ApexActionStatus.cls

```diff
@@ -0,0 +1,13 @@
+public inherited sharing class ApexActionStatus extends Enumeration {
+
+    public static final ApexActionStatus STATUS_WARNING = new ApexActionStatus('WARNING');
```

Comment:
Rename it to WARNING. No need to use the _STATUS prefix, it's already added to the class name. Otherwise, the Status text will appear twice when using it, e.g. ApexActionStatus.STATUS_OK. 
Same for other constants.
```suggestion
    public static final ApexActionStatus WARNING = new ApexActionStatus('WARNING');
```